src/dirs.proj

```diff
@@ -28,7 +28,12 @@
   </PropertyGroup>
 
   <Import Project="$(ToolsDir)packages.targets" Condition="Exists('$(ToolsDir)packages.targets')" />
-  <Target Name="BuidDirPackages" AfterTargets="Build" DependsOnTargets="BuildPackages" />
+  <PropertyGroup Condition="Exists('$(ToolsDir)packages.targets')">
+    <TraversalBuildDependsOn>
+      $(TraversalBuildDependsOn);
+      BuildPackages;
+    </TraversalBuildDependsOn>
+  </PropertyGroup>
 
```

Comment:
Actually, I'll merge this first.  Those targets live in buildtools, so I'll need to bump the version again and re-upload the package.  Will do as a separate change.